TODO.md

```diff
@@ -98,8 +98,8 @@
 
 ## Notifications
 
-- [ ] `useNotifications`
-- [ ] `useUnreadNotifications`
+- [x] `useNotifications`
+- [x] `useUnreadNotifications`
```

Comment:
this one is called `useUnreadNotificationCount`

Reply:
nice spot, updated